src/credentials/GroupDataProviderImpl.h

```diff
@@ -143,9 +143,10 @@ class GroupDataProviderImpl : public GroupDataProvider
     class GroupKeyContext : public Crypto::SymmetricKeyContext
     {
     public:
-        GroupKeyContext(GroupDataProviderImpl & provider) : mProvider(provider) {}
+        GroupKeyContext(GroupDataProviderImpl & provider, bool allocated = false) : mProvider(provider) {}
```

Comment:
What's the point of having the now-unused `allocated` arguments to this constructor signature and the next one?

Reply:
There is no point. Removed.